packages/client/src/sound-board/components/SoundButton.css

```diff
@@ -0,0 +1,51 @@
+:root {
+  --shown: 1;
+}
+
+.button {
+  transform: scale(calc(0.1 + (var(--shown) * 0.9)));
```

Comment:
Jak się szybko scrolluje stronę to ładniejsza animacja jest jak pojawiają się nie od 0 ale o 0.1 wielkości te pastylki, dlatego tak zrobiłem. Do przegadania najwyżej